tests/unit/conftest.py

```diff
@@ -1,28 +1,33 @@
-# pylint:disable=redefined-outer-name
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
 
 import re
+from collections.abc import Iterator
 from random import randint
-from typing import Any, Dict
+from typing import Any
+from unittest.mock import MagicMock
 
-import faker
 import pytest
 from aioresponses import aioresponses
 from aioresponses.core import CallbackResult
+from faker import Faker
+from pytest_mock import MockerFixture
```

Comment:
Same here, could you maybe comment on what this is (used for)?

Reply:
The fixtures in this file were already using these types. I am just importing them to annotate all types.
In tests we always do the following: 
 - All inputs and outputs in fixtures must be annotated
 - all inputs in ``test_`` functions (not the outputs) are annotated

Reply:
Above, those are types from pytest plugins:

- faker: https://github.com/pytest-dev/pytest-faker
- pytest_mock: https://pytest-mock.readthedocs.io/en/latest/